about/index.md

```diff
@@ -1,15 +1,47 @@
 ---
-title: About
+title: About Forseti Security
 ---
-# About
+# {{ page.title }} 
 
-Forseti Security helps you secure your Google Cloud Platform (GCP) environment.
+A community-driven collection of open source tools to improve the security of
```

Comment:
It sounds like Forseti is just a bunch of miscellaneous tools, is this the perception we want to give?